<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vulkan Configurator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="widgetTopLeft" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="group_box_management">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>4</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>135</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>135</height>
            </size>
           </property>
           <property name="title">
            <string>Vulkan Layers Management</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radio_fully">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Layers Fully Controlled by the Vulkan Applications</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radio_override">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Overriding Layers by the Vulkan Configurator</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QWidget" name="widget" native="true">
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="check_box_apply_list">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Apply only to the Vulkan Applications List</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="widget_2" native="true">
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="check_box_persistent">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Continue Overriding Layers on Exit</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="push_button_applications">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>136</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>136</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Edit Applications...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="group_box_configurations">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Vulkan Layers Configurations</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QTreeWidget" name="configuration_tree">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="indentation">
                 <number>5</number>
                </property>
                <property name="columnCount">
                 <number>2</number>
                </property>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerMinimumSectionSize">
                 <number>20</number>
                </attribute>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string notr="true">2</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QPushButton" name="push_button_new">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>New...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="push_button_edit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Edit...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="push_button_remove">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="push_button_find">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Find...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="group_box_launcher">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Vulkan Application Launcher</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QCheckBox" name="check_box_clear_on_launch">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Clear log at launch</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="launcher_loader_debug">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vulkan loader debug messages level used by the VK_LOADER_DEBUG environment variable on launch. &lt;br/&gt;&lt;br/&gt;Levels are:&lt;br/&gt;- error&lt;br/&gt;- warn (warnings)&lt;br/&gt;- info (information)&lt;br/&gt;- debug&lt;br/&gt;- layer (layer information)&lt;br/&gt;- implem (implementation/icd information)&lt;br/&gt;- all (report out all messages)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <item>
            <property name="text">
             <string>none</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Only Errors">error</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>warn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>info</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>layer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>implem</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>all</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="push_button_launcher">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Launch</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="QTextBrowser" name="log_browser">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="push_button_clear_log">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="6">
          <widget class="QTreeWidget" name="launcher_tree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>98</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>98</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">3</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Vulkan Loader Messages:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="group_box_settings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Configuration Settings</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>9</number>
         </property>
         <item row="1" column="0">
          <widget class="QTreeWidget" name="settings_tree">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="animated">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>160</number>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>200</number>
           </attribute>
           <column>
            <property name="text">
             <string notr="true"/>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionVulkan_Info"/>
    <addaction name="actionVulkan_Installation"/>
    <addaction name="separator"/>
    <addaction name="actionRestore_Default_Configurations"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_find_more_layers"/>
    <addaction name="separator"/>
    <addaction name="actionVulkan_specification"/>
    <addaction name="actionVulkan_Layer_Specification"/>
    <addaction name="actionGPU_Info_Reports"/>
    <addaction name="separator"/>
    <addaction name="action_readme"/>
    <addaction name="action_changelog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHistory">
   <property name="text">
    <string>History...</string>
   </property>
  </action>
  <action name="actionSave_Configuration_2">
   <property name="text">
    <string>Save Profile...</string>
   </property>
  </action>
  <action name="actionExit_and_Keep">
   <property name="text">
    <string>Exit and Keep</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVulkan_Info">
   <property name="text">
    <string>Vulkan Info</string>
   </property>
  </action>
  <action name="actionVulkan_Installation">
   <property name="text">
    <string>Vulkan Installation Analysis</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Vulkan Configurator</string>
   </property>
  </action>
  <action name="action_readme">
   <property name="text">
    <string>Vulkan Configurator Readme</string>
   </property>
  </action>
  <action name="actionSet_Test_Application">
   <property name="text">
    <string>Test Environment...</string>
   </property>
  </action>
  <action name="actionCustom_Layer_Paths">
   <property name="text">
    <string>User-Defined Layers Paths...</string>
   </property>
  </action>
  <action name="actionLog_API_Dump">
   <property name="text">
    <string>API Dump...</string>
   </property>
  </action>
  <action name="actionCreate_Custom_Profile">
   <property name="text">
    <string>Create Custom Profile..</string>
   </property>
  </action>
  <action name="actionRestore_Default_Configurations">
   <property name="text">
    <string>Reset To Default</string>
   </property>
  </action>
  <action name="actionVulkan_specification">
   <property name="text">
    <string>Vulkan Specification</string>
   </property>
  </action>
  <action name="actionVulkan_Layer_Specification">
   <property name="text">
    <string>Vulkan Layer Specification</string>
   </property>
  </action>
  <action name="actionGPU_Info_Reports">
   <property name="text">
    <string>Vulkan GPU Info Reports</string>
   </property>
  </action>
  <action name="action_changelog">
   <property name="text">
    <string>Vulkan Configurator Changelog</string>
   </property>
  </action>
  <action name="action_find_more_layers">
   <property name="text">
    <string>Find More Layers!</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
